src/image-enhance/index.rst

```diff
@@ -119,26 +114,23 @@ Run the command below as it applies to the version you have installed
         class Response {
 
             public bool success {get;set;}
-            public Object[] predictions {get;set;}
+            public Enhance[] predictions {get;set;}
 
         }
 
-        class Object {
+        class Enhance {
 
-            public string label {get;set;}
-            public float confidence {get;set;}
-            public int y_min {get;set;}
-            public int x_min {get;set;}
-            public int y_max {get;set;}
-            public int x_max {get;set;}
+            public int base64 {get;set;}
+            public int width {get;set;}
+            public int height {get;set;}

```

Comment:
You don't need this `Enhance` class.  Instead, move it's properties into the `Response` object to replace `public Enhance[] predictions {get;set;}`